libbpf-tools/cpudist.c

```diff
@@ -17,6 +18,8 @@
 static struct env {
 	time_t interval;
 	pid_t pid;
+	char	*cgroupspath;
```

Comment:
nit: Use space after `char`. Same for some other tools. Please follow the original code.

Otherwise looks good to me.

Reply:
Thank you @chenhengqi for the review.
I've updated code to match original format.